Mathlib/Algebra/Homology/QuasiIso.lean

```diff
@@ -51,6 +53,15 @@ lemma quasiIsoAt_iff' (f : K ⟶ L) (i j k : ι) (hi : c.prev j = i) (hk : c.nex
   exact ShortComplex.quasiIso_iff_of_arrow_mk_iso _ _
     (Arrow.isoOfNatIso (natIsoSc' C c i j k hi hk) (Arrow.mk f))
 
+lemma quasiIsoAt_of_retract {f : K ⟶ L} {f' : K' ⟶ L'}
```

Comment:
Would `RetractArrow.quasiIsoAt` make sense? To enable dot notation?

Reply:
I had considered this option, but I am not sure this is the best option, because there are actually three variants of `RetractArrow.quasiIso(At)` lemmas: one for short complexes, one for homological complexes in a given degree (`quasiIsoAt`), and one for homological complexes in all degrees at the same time. The name `RetractArrow.quasiIsoAt` would not be ambiguous, but it would not be clear whether `RetractArrow.quasiIso` is for short complexes or homological complexes. This is the reason why I would rather see these lemmas in their respective `ShortComplex` and `HomologicalComplex` namespaces.